playbooks/collect_activity/stats.py

```diff
@@ -52,8 +77,22 @@
 
     ms = modulestore()
 
+    start_date = args.start_date
+    end_date = args.end_date
+
+    if start_date is None or end_date is None:
```

Comment:
@janimo, this condition will never be true since `--start-date` and `--end-date` are required parameters? I'd recommend having the following logic:
* Both parameters should either be specified together or not at all.
* In the latter case, let the time range default to the last completed month.

Does this make sense?

Reply:
@lgp171188 I overlooked the required=True attribute when copying the code, since there was a default=None setting. The logic you describe was my intention, and should work if the required=True is removed which I have done now. The args should not be required, in case the stats.py script is being called from elsewhere, but also for backwards compat while we develop.